pkg/types/interfacetests/chain_components_interface_tests.go

```diff
@@ -794,6 +794,57 @@ func runQueryKeyInterfaceTests[T TestingT[T]](t T, tester ChainComponentsInterfa
 				}, tester.MaxWaitTimeForEvents(), time.Millisecond*10)
```

Comment:
This is out of scope, but 10ms seems way too fast:
```suggestion
				}, tester.MaxWaitTimeForEvents(), time.Millisecond*500)
```
There is typically little payoff to doing these checks so fast. You risk keeping the CPU so busy that it slows down or cannot even make progress on the test, and the best case payoff is only to finish earlier by the max difference in the durations (assuming parallel tests).